### ISSUE TITLE
[bug] exoscale_compute template attribute not idempotent

### ISSUE THREAD

**Issue by mcanevet** [cab105/terraform-provider-exoscale#2]
Hello,

When I try to create an `exoscale_compute` resource with `template = Linux Ubuntu 16.04 LTS 64-bit`, `terraform apply` yields with:

```
Invalid template: Linux Ubuntu 16.04 LTS 64-bit
```

If I use `template = ubuntu-16.04`, `terraform apply` spawns the VM, but `terraform plan` tells that the current template is `Linux Ubuntu 16.04 LTS 64-bit` and thus will force new resource to change to `ubuntu-16.04`...

```
template:         "Linux Ubuntu 16.04 LTS 64-bit" => "ubuntu-16.04" (forces new resource)
```

**Comment by cab105** [comment]
Thanks for submitting the report.  I will take a look at it.

**Comment by cab105** [comment]
I have resolved this as a part of changeset: 4d1db93cb4905c9e5a1be91bc143b99f9fcd8113
